app/controllers/services_controller.rb

```diff
@@ -27,6 +29,13 @@ def create
 
 private
 
+  def check_authorisation
```

Comment:
this is repeated on each controller, isn't there a way to move this up to the application controller and pass controller name up?